src/Sean_M/DeathPopup/Main.php

```diff
@@ -45,11 +44,12 @@ public function onPlayerDeath(PlayerDeathEvent $event){
       break;
     case EntityDamageEvent::CAUSE_PROJECTILE:
 	if($cause instanceof EntityDamageByEntityEvent){
+            $e = $cause->getDamager();
```

Comment:
getDamager must not be called until it has been confirmed that the event is an EntityDamageByEntityEvent.